huaweicloud/services/ecs/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -108,6 +108,8 @@ func resourceComputeVolumeAttachCreate(ctx context.Context, d *schema.ResourceDa
 		Timeout:      d.Timeout(schema.TimeoutCreate),
 		Delay:        10 * time.Second,
 		PollInterval: 15 * time.Second,
```

Comment:
it's better to decrease the `PollInterval` otherwise users will wait for another 15 seconds.